SmartPlaces.Facilities/lib/OntologyMapper.Mapped/test/OntologyMapper.Mapped.Test.csproj

```diff
@@ -21,12 +21,13 @@
 		  <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
 		</PackageReference>
 		<PackageReference Include="Divergic.Logging.Xunit" Version="4.2.0" />
-		<PackageReference Include="Mapped.Ontologies.Core.Dtdl" Version="1.2.490" />
+		<PackageReference Include="Mapped.Ontologies.Core.Dtdl" Version="1.2.499" />
 		<PackageReference Include="Microsoft.ApplicationInsights" Version="2.21.0" />
 		<PackageReference Update="Microsoft.AspNet.WebApi.Client" Version="5.2.9" />
```

Comment:
Shouldn't this be Include?

Reply:
I thought this was how we needed to set it for the newtonsoft issue?